app/api/schema/tickets.py

```diff
@@ -39,17 +39,17 @@ def validate_quantity(self, data):
         if 'max_order' in data and 'min_order' in data:
             if data['max_order'] < data['min_order']:
                 raise UnprocessableEntity({'pointer': '/data/attributes/max-order'},
-                                          "max-order should be greater than min-order")
+                                          "max-order should be greater than or equal to min-order")
 
         if 'quantity' in data and 'min_order' in data:
             if data['quantity'] < data['min_order']:
                 raise UnprocessableEntity({'pointer': '/data/attributes/quantity'},
-                                          "quantity should be greater than min-order")
+                                          "quantity should be greater than or equal to min-order")
 
         if 'quantity' in data and 'max_order' in data:
             if data['quantity'] < data['max_order']:
                 raise UnprocessableEntity({'pointer': '/data/attributes/quantity'},
-                                          "quantity should be lesser than max-order")
+                                          "quantity should be greater than or equal to max-order")
```

Comment:
Wrong. `quantity should be lesser than or equal to max-order`.

Reply:
But quantity is the total number of tickets.

Reply:
oops! so sorry. You're right. I misunderstood this quantity as the order quantity.